lib/utils/network.js

```diff
@@ -0,0 +1,28 @@
+
+/**
+ *
+ * @param {*} logger
+ * @param {*} hostport
+ * @param {*} protocol
+ * @returns array with bellow properties
+ * 0: protocol
+ * 1: ip address
+ * 2: port
```

Comment:
its actually not clear from this description what this function does. It seems to be filtering so as to return the ip address that the feature server is listening on for a given protocol.  Can you make the description state the purpose more clearly?

Reply:
fixed